### DISCUSSION TITLE
How Often are Released Pushed to the Streamdeck Actions?

### DISCUSSION THREAD

**Discussion by Andywmm9** [nguyenquyhy/Flight-Tracker-StreamDeck#120] [Q&A] [answered]
Should I download and install releases here, or are updates also pushed out inside Streamdeck?

**Comment by rmroc451** [comment]
@nguyenquyhy pushes releases to streamdeck store and https://flightsim.to/file/3178/flight-tracker-streamdeck-plugin when a release is pushed to github.

It may take streamdeck store a bit to sync up 👍

**Comment by nguyenquyhy** [comment, marked as answer]
Stream Deck store requires some manual moderation, so normally it will take a couple of days after I submit the Elgato the package (I do try to send to Elgato the same time I release on GitHub and flightsim.to).

However, Elgato is a bit short on personnel during this holiday period, so update will only resume in 1-2 weeks.
